base/util_testing.go

```diff
@@ -94,16 +94,17 @@ func TestsShouldDropIndexes() bool {
 
 // TestsDisableGSI returns true if tests should be forced to avoid any GSI-specific code.
 func TestsDisableGSI() bool {
-	// FIXME: CBG-813 - Re-enable GSI in integration tests after CB 6.5.1 Beta
-	if true {
-		return true
-	}
 	// Disable GSI when running with Walrus
 	if !TestUseCouchbaseServer() && UnitTestUrlIsWalrus() {
 		return true
 	}
 
-	disableGSI, _ := strconv.ParseBool(os.Getenv(TestEnvSyncGatewayDisableGSI))
+	// Default to disabling GSI, but allow with SG_TEST_DISABLE_GSI=false
```

Comment:
Is there a reason to not call this SG_TEST_USE_GSI, to match the format of SG_TEST_USE_XATTRS and SG_TEST_USE_AUTH_HANDLER?

Reply:
Happy to switch it to conform. Default to false for now to match this behaviour, and flip it to true once GSI tests are stable?

Reply:
Yeah, that makes sense.